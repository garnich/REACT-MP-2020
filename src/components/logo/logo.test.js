import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Logo from './logo';

Enzyme.configure({ adapter: new Adapter() });

describe('Test Logo component', () => {
  it('<Logo> test snapshot', () => {
    const wrapper = shallow(<Logo />)

    expect(wrapper).toMatchSnapshot()
  })
});