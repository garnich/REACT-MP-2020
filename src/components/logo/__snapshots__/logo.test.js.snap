// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Logo component <Logo> test snapshot 1`] = `
<Link
  to="/"
>
  <span
    className="logo"
  >
    <strong>
      netflix
    </strong>
    roulette
  </span>
</Link>
`;
